app/javascript/lib/validation/validate-file-content.js

```diff
@@ -356,12 +356,16 @@ export async function parseFile(file, fileType, fileOptions={}, sizeProps={}) {
     delimiter: null,
     isGzipped: null
   }
+
   const parseResult = { fileInfo, issues: [] }
 
   try {
     fileInfo.isGzipped = await validateGzipEncoding(file)
     // if the file is compressed or we can't figure out the compression, don't try to parse further
-    if (fileInfo.isGzipped || !PARSEABLE_TYPES.includes(fileType)) {
+    if (
+      !PARSEABLE_TYPES.includes(fileType) ||
+      file.size > sizeProps?.fileSizeTotal && fileInfo.isGzipped // Avoids bug in sync gzip CSFV
```

Comment:
what bug? can you clarify?

Reply:
I've refined and extended this comment in d5b0b00.  

Thanks for prompting another look here -- I found an actual bug in this updated CSFV handling for "large" (> 50 MiB) gzipped files.  That's fixed now too.